app/models/attendance.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Attendance < ApplicationRecord
+  belongs_to :attendee, class_name: 'User', foreign_key: 'attendee_id'
```

Comment:
guys we don't need foreign keys here at all. Attendance has no keys on `User` nor `Event`

Reply:
Fixed by removing unnecessary `foreign_key:`